packages/vite/src/node/cli.ts

```diff
@@ -52,23 +52,42 @@ function cleanOptions(options: GlobalCLIOptions) {
   return ret
 }
 
+export const SHORTCUTS = [
```

Comment:
mh 🤔 
due to we need to export this array, we could add an interface for this (or reuse one if there exists one)
or we could write `as const` behind the closing array bracket on line 77 to make the array readonly

Reply:
Could just write an interface for it I think